app.js

```diff
@@ -171,27 +176,43 @@ client.on(`message`, (message) => {
  * Action: Log the metadata about the message, and the message itself
  */
  client.on(`messageDelete`, message => {
+  for(var i = 0; i < logBlackList.length; i++){
+    if(logBlackList[i]===message.channel.name){
+        return;
+    }
+}
   attachments = message.attachments.array().length!=0 ? "Yes" : "No";
+  
   util.logToServer(`The following message was deleted:
     Id: ${message.id}
-    Author: ${message.author}
+    Author: ${message.author.username}
```

Comment:
We need the actual author link - not the username. Especially important with people changing their names.

Reply:
Done.